overrides/customHttp.yml

```diff
@@ -23,6 +23,7 @@ customHeaders:
           script-src 
           'self' 
           'sha256-7tJzJRhCSII909o84m4q85UWUc5EDMrrjsQXbeH+qlc='
+          'sha256-nWKjNpDy9BIIH8p69UATrM+dYfeHm3RCw7s03nOoDC0=' # Script for GTM tag
```

Comment:
should we be removing the old one? i think this is a back-compat change iirc as all sites invoke this script at the point of build, which means that every subsequent build is safe (will use new gtm, old one never worked) + existing builds no longer need gtm (build script alr ran)

i think it's easy to do this now rather than having to clear it next time, where we no longer have it fresh on our minds

Reply:
Unfortunately i can't confirm what the old sha is for anymore :/ Hence the comment on the new one so that we're aware of this in future

Reply:
okay nvm verified that this was for the old tag

I removed the old sha whitelist and rebuilt a-test-v4 - encountered error
rebuilt a-test-v4 on a template which didn't make use of the old gtm tag - no error